<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDBsourceDialog</class>
 <widget class="QDialog" name="NewDBsourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据源属性</string>
  </property>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>61</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheck">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>创建与数据组对应的表</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_pCheckShow">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>显示表</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>OkButton</tabstop>
  <tabstop>CancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
